src/lib.rs

```diff
@@ -123,19 +123,30 @@
 //! The letter case is not significant for the logging level names; e.g.,
 //! `debug`, `DEBUG`, and `dEbuG` all represent the same logging level.
 //!
+//! There is also a pseudo logging level, `OFF`, which may be specified to
```

Comment:
Since the "canonical" (most often referred to) form is lowercase, maybe this should be lowercase as well.

Reply:
Okay. I have only limited experience with `env_logger`; my experience with other loggers in other language ecosystems has conditioned me to recognize the upper case as something akin to either a global constant or the name of a global variable name. But I'm not fixated on it  :-)

As I'm tweaking things for my rebase, I'll try to work in a mention that lowercase is preferred.

Reply:
Oh, I wasn't saying that it should be preferred. Just that the docs most often use the lowercase form. I was suggesting

```suggestion
//! There is also a pseudo logging level, `off`, which may be specified to
```